downstream/assemblies/eda/assembly-eda-projects.adoc

```diff
@@ -6,6 +6,12 @@ Projects are a logical collection of rulebooks.
 They must be a git repository and only http protocol is supported. 
 The rulebooks of a project must be located in the path defined for {EDAName} content in Ansible collections: `/extensions/eda/rulebooks` at the root of the project.
 
+[NOTE]
+====
+* Do not link a project to an empty repository. Linking a project to an empty repository causes synchronization errors.
```

Comment:
I believe this is no longer the case.  See: https://github.com/ansible/eda-server/pull/988

Reply:
That note was added separately for another jira. I can look into this and make sure it gets removed if that's the case.